src/role-access/containers/allocate-role/choose-role/choose-role.component.ts

```diff
@@ -91,6 +81,18 @@ export class ChooseRoleComponent implements OnInit, OnDestroy {
     }
   }
 
+  private getOptions(roles: Role[]): OptionsModel[] {
+    const options = [];
```

Comment:
could simplify this code 
from
`
roles.forEach((jr) => {
      const thisOption: OptionsModel = {
        optionId: jr.roleId,
        optionValue: jr.roleName
      };
      options.push(thisOption);
    });
    return options;`
to 

`return roles.map(jr => ({optionId: jr.roleId, optionValue: jr.roleName}))`;

Reply:
Done